libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4548,6 +4560,12 @@ void GCS_MAVLINK::handle_message(const mavlink_message_t &msg)
         break;
 #endif
 
+#if AP_AIS_ENABLED
+    case MAVLINK_MSG_ID_AIS_VESSEL:
+        handle_ais_message(msg);
+        break;
+#endif
```

Comment:
Right, but they should.

I've noticed in my rebasing the closing comments really help resolve conflicts as the resolution doesn't try to see the closing `#endif` as a common line between the two patches.  *With* the comment you can usually just accept both patches, without the comment you have to go and manually add another `#endif` to the code.

Reply:
ok, fair enough.. done